### DISCUSSION TITLE
Retries in playlists shouldn't count towards total attempts if no objects are hit

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#15386] [Ideas]
I've realized that on playlists that have a limited amount of attempts, it isn't uncommon for people to be either confused and/or frustrated regarding how their number of total attempts is so high. I believe that it is because people assume that a play doesn't count as an attempt unless you actually hit a note, but this isn't the case. If you load into a map, it will automatically count as an attempt even if you quit out of the map before any objects are even visible.

I personally believe that this should be changed as I wouldn't consider loading into a map to be an attempt if you're not actually hitting any objects in the first place. If you load into a map and start actually hitting objects, I would consider that to be an attempt that should be counted and I don't think that anyone really disagrees with this.

**Comment by peppy** [comment]
For sure. Similar to stable there will likely be a number of conditions which need to be met to qualify for score submission. Right now no such conditions exist.
